readme.md

```diff
@@ -93,6 +93,13 @@ Default: [Only in development](https://github.com/sindresorhus/electron-is-dev)
 
 Force enable or disable the `Inspect Element` menu item.
 
+#### showServices
+
+Type: `boolean`<br>
+Default: `true`
+
+Show the system `Services` submenu on macOS.
+
```

Comment:
Can you add a note about it not being exactly like the native one and link to the Electron issue?